virtualizarr/xarray.py

```diff
@@ -41,9 +41,52 @@ def open_virtual_dataset(
     drop_variables: Iterable[str] | None = None,
     loadable_variables: Iterable[str] | None = None,
     decode_times: bool | None = None,
-    cftime_variables: Iterable[str] | None = None,
```

Comment:
@TomNicholas what do you think about removing this unused parameter?

Reply:
Get it out of here. As we're breaking some part of the API we should just do a clean break of everything old in one go.